packages/core/src/build/configAndIndexingFunctions.test.ts

```diff
@@ -51,7 +52,7 @@ test("buildConfigAndIndexingFunctions() builds topics for multiple events", asyn
     options,
   });
 
-  expect(sources[0].criteria.topics).toMatchObject([
+  expect((sources[0] as LogSource).criteria.topics).toMatchObject([
```

Comment:
This is because `sources` is coming from the test context, it's coming from the `buildConfigAndIndexing` which isn't as strict. `BlockSource` doesn't have a "topics" property on the criteria